LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua

```diff
@@ -121,10 +185,23 @@ local function CreateLabelAndContainerControl(parent, controlData, controlName)
         label:SetAnchor(TOPRIGHT, container, TOPLEFT, 5, 0)
     end
 
+    local anchorFAQX = 0
+    local anchorFAQPos = LEFT
+    local anchorFAQRelativePos = LEFT
+    if label.GetTextWidth then
```

Comment:
Not sure what the purpose of this check is? Label should always be a label control, so it either has this method or not. Or is it a newly added function on the pts?

Reply:
During my tests there was one circumstance where the GetTextWidth function did not exist on the "label" variable. Maybe I managed to pass some other control than a real label to it or it was a submenu's label (which got the function as well). Wil ldo some more tests.